Apps/Database/src/Constants/CommentEntryType.cs

```diff
@@ -38,6 +43,6 @@ public static class CommentEntryType
         /// <summary>
         /// The code representing Encounter.
         /// </summary>
-        public const string Encounter = "ENC";
+        public const string Encounter = "Enc";
```

Comment:
This is existing but it should likely be converted to an enum with value convertors on the dbcontext.  We can raise a bug on this.

Reply:
We have a mix of enums and static classes on the DB constants. Not sure which one we should go forth, but I think we should change them all.

Reply:
created 9955 to track it